src/containerapp/azext_containerapp/tests/latest/test_containerapp_up.py

```diff
@@ -35,3 +35,6 @@ def test_containerapp_up_image_e2e(self, resource_group):
         url = url if url.startswith("http") else f"http://{url}"
         resp = requests.get(url)
         self.assertTrue(resp.ok)
+
+        self.cmd(f"containerapp up --image {image} --environment {env_name} -g {resource_group} -n {app_name} -l {TEST_LOCATION.upper()}")
```

Comment:
This case doesn't validate anything after this up command?

Reply:
--location is upper(), before it will throw exception:
Environment env57ex5rpqfftbdgu5epma2 already exists in resource group clitest.rgi3r4jbaoqhhjdnxicrxm7umtbm4qwazpsvfzakbxyigjwdp7hke56kykselffqrwe on location northcentralus, cannot change location of existing environment to NORTHCENTRALUS.